src/script/cryptography/CryptographyRepository.ts

```diff
@@ -266,8 +294,8 @@ export class CryptographyRepository {
       isExternal && eventData.data.length > Config.getConfig().MAXIMUM_MESSAGE_LENGTH_RECEIVING;
     if (genericMessageIsTooBig || externalMessageIsTooBig) {
       const error = new ProteusErrors.DecryptError.InvalidMessage('The received message was too big.', 300);
-      const errorEvent = z.conversation.EventBuilder.buildIncomingMessageTooBig(event, error, error.code);
-      return Promise.resolve(errorEvent);
+      const errorEvent = window.z.conversation.EventBuilder.buildIncomingMessageTooBig(event, error, error.code);
```

Comment:
`EventBuilder` is already written in TypeScript and mostly static. Can you import it?